src/cata_tiles.cpp

```diff
@@ -2757,6 +2758,21 @@ bool cata_tiles::draw_zone_mark( const tripoint &p, lit_level ll, int &height_3d
     return false;
 }
 
+bool cata_tiles::draw_zombie_revival_indicators( const tripoint &pos, const lit_level /*ll*/,
+        int &/*height_3d*/, const bool ( &invisible )[5] )
+{
+    if( !invisible[0] && g->m.could_see_items( pos, g->u ) &&
```

Comment:
Could you also add `item_override.find( p ) == item_override.end()` to the conditions? It ensures that revival indicators are not shown on debug mapgen preview tiles.

Reply:
Thanks, done.